app/models/activity.rb

```diff
@@ -20,10 +20,12 @@ class Activity < ApplicationRecord
   enum level: {
     fund: "fund",
     programme: "programme",
+    project: "project",
   }
 
   scope :funds, -> { where(level: :fund) }
   scope :programmes, -> { where(level: :programme) }
+  scope :projects, -> { where(level: :project) }
```

Comment:
As we have `level` as an enum I think we get this (and the 2 pre-existing) scopes for free?

https://api.rubyonrails.org/v5.2.4.1/classes/ActiveRecord/Enum.html